features/cinema/CinemaHall.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+
+const CinemaHallSchema = new mongoose.Schema({
+  numberOfHall: Number,
+  numberOfSeats: Number,
```

Comment:
maybe just 'columns'

Reply:
Not columns, because i would like I would like to specify the number of seats here. Number of columns i will get from operation (seats/rows).